### PR TITLE
Use constant instead of strings and remove useless statement in SBT

### PR DESCRIPTION
**Pull request by joan38**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -110,7 +110,6 @@ lazy val core = project
     metaMacroSettings,
     moduleName := "scalafmt-core",
     test in assembly := {
-      (test in Test).value
```
Comment (olafurpg) [inline build.sbt]: The task runs, so the SBT warning is wrong, but I ever run root/assembly anymore so I'm OK with dropping this line.

**Comment by olafurpg** [comment]
LGTM 👍 Thanks :)

File: build.sbt (hunk 1 above)
Comment (joan38) [inline build.sbt]: Do you want to drop the whole setting then?
```
test in assembly := { ... }
```

Comment (olafurpg) [inline build.sbt]: Yes, that would be great.

**Pull request merged**